sync/class.jetpack-sync-actions.php

```diff
@@ -41,6 +46,11 @@ static function init() {
 
 		// On jetpack registration
 		add_action( 'jetpack_site_registered', array( __CLASS__, 'schedule_full_sync' ) );
+
+		// Schedule a job to send DB checksums once an hour
+		if (! wp_next_scheduled ( 'jetpack_send_db_checksum' )) {
+			wp_schedule_event( time(), 'hourly', 'jetpack_send_db_checksum' );
```

Comment:
My only concern here is that we might be creating some sort of performance hit. Do we have any data how fast we expect the checksum to be calculated when a site has 50,000 comments for example.